src/main/resources/openapi.json

```
@@ -1,286 +1,1279 @@
 {
-  "openapi": "3.0.2",
-  "info": {
-    "title": "HTTP Kafka bridge",
-    "version": "1.0.0"
-  },
-  "paths": {
-    "/topics/{topicname}": {
-      "post": {
-        "description": "Write to topic",
-        "operationId": "send",
-        "responses": {
-          "200": {
-            "description": "OK",
-            "content": {
-              "application/json": {}
-            }
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "topicname",
-          "in": "path",
-          "description": "Topic name",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
-        }
-      ]
-    },
-    "/topics/{topicname}/partitions/{partitionid}": {
-      "post": {
-        "operationId": "sendToPartition",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "topicname",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Strimzi HTTP Bridge for Apache Kafka",
+        "description": "The Strimzi HTTP Bridge for Apache Kafka provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol. The HTTP Bridge can be deployed on bare metal or in a container running on Kubernetes or OpenShift platform.",
```

How the bridge might be deployed is completely orthogonal to the API it provides, which is what this json is describing. So I suggest we drop the final sentence.

```suggestion
        "description": "The Strimzi HTTP Bridge for Apache Kafka provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol.",
```